src/components/featuredContent/featuredContent.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  FeaturedContent,
+  Button,
+  Heading,
+} from "@nypl/design-system-react-components";
+import React from "react";
+import featuredContentData from "@/data/featuredContentData";
+import { FeaturedContentData } from "@/types/FeaturedContentData";
+
+const FeaturedContentComponent = ({ testRandomNumber }) => {
+  const randomNumber =
+    testRandomNumber !== undefined
+      ? testRandomNumber
+      : Math.floor(Math.random() * 2);
+  const data: FeaturedContentData = featuredContentData[randomNumber];
+  return (
+    <FeaturedContent
+      data-testid={`featured-content-${randomNumber}`}
+      isFullWidth={true}
+      textContent={
+        <div>
+          <Heading level="h2" overline={data.overline}>
+            {data.heading}
+          </Heading>
+          <p>{data.text}</p>
+
+          <a href={data.link} target="_blank" rel="noopener noreferrer">
```

Comment:
If you use a DS `Link` component, you can make it look like a button: https://nypl.github.io/nypl-design-system/reservoir/v2/?path=/story/components-navigation-link--with-controls&args=type:buttonPrimary

That's better for accessibility. The general rule is links navigate to a new page and buttons are interactive and update the current page. In this case, we want to go to a new page but the UX design is for a button, so we have to reconcile that.